### PR TITLE
Add BigQuery Remote Vision Function code sample

### LINKED ISSUE #8425: Add BigQuery Remote Function code samples for Unstructured Data Analytics
BigQuery Remote Function code samples are currently inline in documentation.
https://cloud.google.com/bigquery/docs/reference/standard-sql/remote-functions#sample_code

Instead of adding more code samples inline, new code samples for Unstructured Data Analytics are preferred to be added in repo. Because the new samples use multiple Google Cloud Platform products, it is preferred to add the new samples in this repo instead of to [googleapis/python-bigquery](https://github.com/googleapis/python-bigquery).

### PR DESCRIPTION
**Pull request by autoerr**
## Description

First sample for #8425

## Checklist
- [x] Before submitting a pull request, please open an issue for discussion if you are not associated with Google.
- [x] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md)
- [x] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#readme-file)
- [x] **Tests** pass:   `nox -s py-3.9` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [x] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [x] Please **merge** this PR for me once it is approved.

### CODE REVIEW

File: bigquery/remote-function/vision/noxfile_config.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# TEST_CONFIG_OVERRIDE copied from the source of truth:
+# https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/noxfile_config.py
+
+TEST_CONFIG_OVERRIDE = {
+    # You can opt out from the test for specific Python versions.
+    "ignored_versions": ["2.7", "3.6"],
```
Comment (engelke) [inline bigquery/remote-function/vision/noxfile_config.py:20]: The README says the samples are compatible with 2.7, so shouldn't they be tested under that version, too?

File: bigquery/remote-function/vision/noxfile_config.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    # Old samples are opted out of enforcing Python type hints
+    # All new samples should feature them
+    "enforce_type_hints": False,
```
Comment (engelke) [inline bigquery/remote-function/vision/noxfile_config.py:23]: Since this is a new sample, this should be True.

**Review by engelke — APPROVED** [review]
LGTM, see comments in nox file, but don't consider them blocking if you disagree with them.

File: bigquery/remote-function/vision/noxfile_config.py (hunk 1 above)
Comment (autoerr) [reply bigquery/remote-function/vision/noxfile_config.py:20]: Updated the README to be 3.6+ as the underlying API client and framework will likely drop support for Python 2.

File: bigquery/remote-function/vision/noxfile_config.py (hunk 2 above)
Comment (autoerr) [reply bigquery/remote-function/vision/noxfile_config.py:23]: Done.

**Review by dandhlee — COMMENTED** [review]
Could you add yourself as the owner for the sample here? https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/CODEOWNERS

A team is preferred if there is one!

@shollyman, could you take a look as BQ owner?

**Review by dandhlee — APPROVED** [review]
Thank you! Approving for CODEOWNERS.

Adding `do not merge` until BQ approval is obtained.

**Review by shollyman — APPROVED** [review]

**Pull request merged**